examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1184,7 +1189,7 @@ class GLTFWriter {
 
 		const textureDef = {
 			sampler: this.processSampler( map ),
-			source: this.processImage( map.image, map.format, map.flipY )
+			source: this.processImage( map.image, map.format, map.flipY, map.userData.mimeType )
```

Comment:
Note this will result in invalid files if the input was WebP; we'd need to append `EXT_texture_webp`.

Reply:
Indeed; fodder for a next PR?

Reply:
If you're planning to make that PR in the next week or two, sure! Otherwise I'd just limit the output to PNG or JPEG here for now.

Reply:
sure, I can limit it for now.

Reply:
Thanks! Just hoping to avoid having an interim span of time where we output invalid files. :)

Reply:
Done.